sdk/fleet/Azure.ResourceManager.ContainerServiceFleet/src/autorest.md

```diff
@@ -31,6 +31,7 @@ rename-mapping:
   SkipTarget: ContainerServiceFleetSkipTarget
   TargetType: ContainerServiceFleetTargetType
   UpgradeChannel: ContainerServiceFleetUpgradeChannel
+  GenerateResponse: GenerateResult
```

Comment:
this name is kind of too generic, could we add something to make it more specific to let people know which kind of generate result this is?